packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -633,17 +633,7 @@ export class AtomicSearchBox {
     this.searchBoxAriaMessage = '';
   }
 
-  private onSuggestionMouseOver(
-    item: SearchBoxSuggestionElement,
-    side: 'left' | 'right',
-    id: string
-  ) {
-    const thisPanel = side === 'left' ? this.leftPanelRef : this.rightPanelRef;
```

Comment:
Does this imply that if instant results are being displayed in the right panel, then hovering a query suggestion in the left panel would not update the instant results? I.e., you'd have to use the arrows to set focus on those?

If so, I'm not convince this is the behavior we want in this specific case